### DISCUSSION TITLE
Block/Unblocking Realm Question

### DISCUSSION THREAD

**Discussion by greenmg2003** [ppy/osu#16926] [Q&A] [answered]
When I pressed "Block Realm", why do I feel like the realm isn't blocked? Like, the button "Unblock Realm" is unable to be clicked.
Okay, just to be more, after I pressed Block Realm, the screen froze, and then after unfreezing, probably nothing changed
Video:  https://drive.google.com/file/d/1Y6QgoCK90lROKjsNUuHMCXL1S9QvU7iv/view?usp=sharing
(The video is 15.8MB, so I have to upload it on Drive)
Log file:
[database.log](https://github.com/ppy/osu/files/8103429/database.log)
[runtime.log](https://github.com/ppy/osu/files/8103431/runtime.log)
Let me know if you need more files

**Comment by peppy** [comment, marked as answer]
Don't use the button. It's there for us, not you.

**Reply by greenmg2003** [reply]
Ah thank you
